test/e2e/framework/util.go

```diff
@@ -388,6 +389,39 @@ func RunIfSystemSpecNameIs(names ...string) {
 	skipInternalf(1, "Skipped because system spec name %q is not in %v", TestContext.SystemSpecName, names)
 }
 
+// Run a test container to try and contact the Kubernetes api-server from a pod, wait for it
+// to flip to Ready, log its output and delete it.
+func runKubernetesServiceTestContainer(c clientset.Interface, ns string) {
+	path := "test/images/clusterapi-tester/pod.yaml"
```

Comment:
I couldn't find it as well. I'm trying to check when it was removed and why.

Reply:
To me `runKubernetesServiceTestContainer` looks like dead code that cannot work because of the missing `test/images/clusterapi-tester/pod.yaml`. As no-one seems to have noticed, I propose to delete the entire function.